dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -1132,10 +1132,22 @@ std::vector<BlobFileId> BlobStore<Trait>::getGCStats()
             }
 
             // Check if GC is required
-            if (stat->sm_valid_rate <= config.heavy_gc_valid_rate)
+            // Raft related data contains raft log and other meta data.
+            // Raft log is written and deleted in a faster pace which is not suitable for full gc.
```

Comment:
The GC behavior should be able to controlled by configurations, instead of directly put here.

Reply:
what kind of `GC behavior` do you mean, and what kind of configurations do you want to add?

Reply:
Fixed.